src/flaskapp/schemas.py

```diff
@@ -0,0 +1,63 @@
+from functools import wraps
```

Comment:
Rather than `schemas.py` we can change this file to `auth.py`

Reply:
and may merge want to `lcs.py` with this file.

Reply:
great idea! resolved with 78da402

Schema validation is something we should consider - tracking separately in https://github.com/HackRU/teamRU/issues/17.